src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -687,6 +688,47 @@ public override ImmutableArray<string> TupleElementNames
         }
 
         /// <summary>
+        /// Get the fields for the tuple's elements (in order and cached).
+        /// </summary>
+        public ImmutableArray<FieldSymbol> TupleElementFields
```

Comment:
Yes, i noticed the code and I needed similar functionality so I just copied so that it'd be easier to merge.